### ISSUE TITLE
inline missing

### ISSUE THREAD

**Issue by KuDeSnik33ra** [artem-ogre/CDT#3]
Theese functions should be marked inline - otherwise, compiling projects with multiple cpp files, including CDT, will fail.
CDTUtils.hpp,99 
```
Edge::Edge(VertInd iV1, VertInd iV2)
    : m_vertices(
          iV1 < iV2 ? std::make_pair(iV1, iV2) : std::make_pair(iV2, iV1))
{}

bool Edge::operator==(const Edge& other) const
{
    return m_vertices == other.m_vertices;
}

VertInd Edge::v1() const
{
    return m_vertices.first;
}

VertInd Edge::v2() const
{
    return m_vertices.second;
}

const std::pair<VertInd, VertInd>& Edge::verts() const
{
    return m_vertices;
}
```

**Comment by artem-ogre** [comment]
What's the compilation error? Do you have `CDT_USE_AS_COMPILED_LIBRARY` defined?

**Comment by artem-ogre** [comment]
I was able to reproduce it.

**Comment by KuDeSnik33ra** [comment]
Hello, thanks!

**Comment by artem-ogre** [comment]
Thanks a lot for reporting the issue!
Fixed in: f53d788
